src/http/request.c

```diff
@@ -31,8 +31,8 @@
 #define DEBUG_LEVEL 5
 #include <re_dbg.h>
 
-#ifndef VERSION
-#define VERSION "???"
+#ifndef RE_VERSION
+#define RE_VERSION "???"
```

Comment:
The version string is re-defined here ?

Reply:
Yes this looks strange, better to include `re_sys.h`

Reply:
Yes, I add a commit.